_quarto.yml

```diff
@@ -32,14 +32,14 @@ website:
       align: left
       collapse-level: 1
       contents:
-      - about/index.md
+      - about/index.qmd
       - about/principles.qmd
       - text: "History"
-        href: about/history/index.md
+        href: about/history/index.qmd
       # - text: "Mission"
       #   href: about/mission.md
```

Comment:
Do we want to mop up all of them? It's just that there are two or three missing from your edits.

Reply:
If I'm doing the search thing right, then we also have a code of conduct document which is .md (referenced in the readme file), and the nnf-instructions file which is mentioned in history/about/index.

In usage/index also have a few links to files that are still md.

Reply:
You got it! Yea, some files will need to be in `.md` format since they are discoverable by GitHub that way, like the code of conduct. The usage docs I am deleting in #278. And good point about the `nnf-instructions`, I misread that as the application, which I didn't want to touch.